plugin/core/diagnostics_manager.py

```diff
@@ -9,6 +10,19 @@
 T = TypeVar('T')
 
 
+def by_location(diagnostic: Diagnostic) -> Tuple[int, int]:
+    line = diagnostic.get("range", {})["start"]["line"]
+    column = diagnostic.get("range", {})["start"]["character"]
+    return line, column
+
+
+def by_severity(diagnostic: Diagnostic) -> Tuple[int, int, int]:
+    severity = diagnostic.get("severity", -1)
```

Comment:
Severity can be omitted, and according to the specs it is up to the client how to interpret the diagnostic in that case.
`-1` here would show such diagnostics at the top (above errors), maybe we rather should move them to the bottom instead?

Reply:
I changed it to list them at the bottom, which I think is a better order.